src/main/java/entities/Item.java

```diff
@@ -1,28 +1,31 @@
 package entities;
 
```

Comment:
Added facade design pattern to item class for reading and generating barcodes to delegate barcode tasks to other classes.